### DISCUSSION TITLE
Pdf embed signature

### DISCUSSION THREAD

**Discussion by pmjam** [MatthiasValvekens/pyHanko#227] [Q&A]
Hi, I'm facing an issue when I try to attach a signature for a second time. The first time I click on the button to sign and attach the signature, it works. But, when I try to do it for the second time on another pdf, it doesn't, it says that the integrity check failed. 

Sorry, about that. It was my mistake.

**Comment by MatthiasValvekens** [comment]
Locking this, as it seems like there's nothing further to discuss after your edit :)
